src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py

```diff
@@ -418,3 +418,24 @@
   - name: Run an existing workflow in your github repository
     text: az postgres flexible-server deploy run --action-name testAction --branch userBranch
 """
+
+helps['postgres flexible-server backup'] = """
+type: group
+short-summary: Manage flexible server backups.
+"""
+
+helps['postgres flexible-server backup list'] = """
+type: command
+short-summary: List all the backups for a given server.
+examples:
+  - name: List all backups for 'testsvr'.
+    text: az postgres flexible-server backup list -g testgroup -n testsvr
+"""
+
+helps['postgres flexible-server backup show'] = """
```

Comment:
Why `postgres` doesn't support `backup create`? If customer can't create a backup, then what's the meaning for `postgres backup show/list`?

Reply:
It'll be available next month I think, but I was requested to add those two operations for PostgreSQL.